app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1504,8 +1504,14 @@ body {
   color: #FFFFFF;
 }
 
+a {
+  font-family: 'Cardinal', Arial, sans-serif;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.